include/triton/common/logging.h

```diff
@@ -73,9 +76,48 @@ class Logger {
   // Get the logging format.
   Format LogFormat() { return format_; }
 
+  // Get the logging format as a string.
+  std::string LogFormatString()
+  {
+    switch (format_) {
+      case Format::kISO8601:
+        return "ISO8601";
+      case Format::kDEFAULT:
+        return "default";
+      default:
+        return "Invalid format";
+    }
+  }
+
   // Set the logging format.
   void SetLogFormat(Format format) { format_ = format; }
 
+  // Get the log output file name.
+  const std::string& LogFile() { return filename_; }
+
+  // Set the log output file.
```

Comment:
May be worth adding a comment to specify that the return value is an empty string upon success, else error string. Not sure if you did this elsewhere, but saves reading through the function to see how to error check (or whether error checking happens here).